Automattic/wp-calypso
# Purchase Modal: add README and move files

## Proposed Changes

* Adds a README.md file to explain the component usage and props. [Click here](https://github.com/Automattic/wp-calypso/blob/refactor/move-purchase-modal/client/my-sites/checkout/purchase-modal/README.md) to view the file.
* Move the `purchase-modal` directory from `client/my-sites/checkout/upsell-nudge` to `client/my-sites/checkout/` as it is no longer tied to upsell nudge.

## Testing Instructions



* This PR does not contain any functional changes.
* Ensure that you have a saved credit card.
* Go to `/plugins/<site slug>` for a site on the free plan.
* Confirm that you can see the 1-click checkout modal.

## Pre-merge Checklist
